README.md

```diff
@@ -94,472 +92,21 @@ Qubes uses SaltStack internally for VM provisionining and configuration manageme
 
-1. Verify the tag in the [qubes-template-securedrop-workstation](https://github.com/freedomofpress/qubes-template-securedrop-workstation) repository: `git tag -v VERSION` and ensure the tag is signed with the official release key.
-2. `git checkout VERSION`
-3. Rebuild template following documentation in [qubes-template-securedrop-workstation](https://github.com/freedomofpress/qubes-template-securedrop-workstation).
-4. sha256sum the built template (and store hash in the build logs/commit message).
-5. Commit unsigned template for historical purposes.
-6. Sign template RPM with test key (rpm --resign <file>) (see Signing section below).
-7. Commit signed template.
-8. Push those two commits to a PR in [securedrop-workstation-dev-rpm-packages-lfs](https://github.com/freedomofpress/securedrop-workstation-dev-rpm-packages-lfs/). Make the PR.
-9. Upload build logs directly to the [build-logs](https://github.com/freedomofpress/build-logs) repository in the workstation directory.
-10. Upon merge of the PR into [securedrop-workstation-dev-rpm-packages-lfs](https://github.com/freedomofpress/securedrop-workstation-dev-rpm-packages-lfs/), the template will be deployed to `yum-test.securedrop.org`.
-11. Test template.
-12. Once template is sufficiently tested, remove test sig: `rpm --delsign <file>`.
-13. Verify unsigned template sha256sum from build logs/commit message.
-14. Sign template with prod key: `rpm --resign <file>`
-15. Push commit to a branch in the [securedrop-workstation-prod-rpm-packages-lfs](https://github.com/freedomofpress/securedrop-workstation-prod-rpm-packages-lfs/) repository. Make a PR.
-16. Upon merge to master, ensure that changes deploy to `yum.securedrop.org` without issue.
-
-### Signing binaries/packages
+The project is currently in a closed beta, and we do not recommend installing it for production purposes. If you are participating in our beta program, please consult our *[end user documentation](https://workstation.securedrop.org)* for detailed setup instructions, and do not hesitate to reach out for assistance.
 
-#### Debian packages
+To install a development version (using test data on the server and a test encryption key to decrypt it), in summary, you will need to:
 
-The apt repository Release file will be signed, containing checksums of the debs.
+1. Ensure you have an up-to-date SecureDrop server or staging environment
+2. Apply all available system updates in Qubes OS
+3. Clone this repository into a development VM
+4. Create a `config.json` and `sd-journalist.sec` (Submission Private Key) appropriate for your environment
+4. Copy the contents from your development VM into `~/securedrop-workstation` in `dom0`
+5. In `dom0`, export `SECUREDROP_DEV_VM` (name of your dev VM) and `SECUREDROP_DEV_DIR` (full path to repo checkout in dev VM) and run `make clone`
+5. Run `make dev` in `dom0` to provision a development environment.
```

Comment:
This "quickstart" style summary is new to this PR. I'm open to not including it if folks feel it's more likely to confuse than to help, but I figure if anything belongs at the README level, it's a *summary* of how to get the thing to run, to the extent such a thing is possible.

Reply:
I think this is much clearer now